modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/constants/DDMFormConstants.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.dynamic.data.mapping.form.web.internal.constants;
+package com.liferay.dynamic.data.mapping.constants;
```

Comment:
@carolmariaabb, why did you decide to do it? Can you please clarify?

Reply:
@natocesarrego because I need these same constants in the `form-web` module and in the `form-field-type` module.

Reply:
Makes sense. Thanks for the clarification. 😉